packages/calcite-components/src/components/tree-item/tree-item.tsx

```diff
@@ -289,10 +288,23 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
 
     return (
       <Host
-        aria-checked={this.selectionMode === "ancestors" ? toAriaBoolean(this.selected) : undefined}
+        aria-checked={
```

Comment:
Awesome! 👍 

Should we have an e2e test for this? Could be done as follow up issue due to time constraints.

Reply:
I added e2e tests for this.